### PR TITLE
Use one shared informer for all libvirt lifecycle events

### PR DESCRIPTION
**Pull request by rmohr**
With the exposed functions in the new go bindings it is not possible to
register multiple listeners for the same events. Therefore use one
shared informer to inform all interested party via that one.

### CODE REVIEW

**Comment by rmohr** [comment]
@mpolednik That should fix the events. Want to to a review?

**Comment by rmohr** [comment]
Basic functional test will follow shortly ...

File: cmd/virt-handler/virt-handler.go [hunk 1]
```diff
@@ -20,6 +19,7 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-handler"
 	"kubevirt.io/kubevirt/pkg/virt-handler/virtwrap"
 	virtcache "kubevirt.io/kubevirt/pkg/virt-handler/virtwrap/cache"
+	"time"
```
Comment (mpolednik) [inline cmd/virt-handler/virt-handler.go]: seems like the auto import ordering tools battling eachother?

**Comment by rmohr** [comment]
retest this please

**Pull request merged**